format.go

```diff
@@ -42,7 +43,7 @@ func writeFolded(w io.Writer, s map[string]int) error {
 	return nil
 }
 
-func toPprof(s map[string]int, hz int) *profile.Profile {
+func toPprof(s map[string]int, hz int, startTime time.Time) *profile.Profile {
```

Comment:
It seems like this is not used? Am I missing something?

Reply:
Yeah, looks like this patch is passing the timestamp to the right place, but stops short of actually applying it to the profile.

Reply:
My bad, I had the changes uncommitted locally. Pushed them now!